public/js/notes.js

```diff
@@ -11,24 +11,24 @@ function updateNote() {
             <th scope="row">${index + 1}</th>
             <td>${element[0]}</td>
             <td>${element[1]}</td>
-            <td><button id="dBtn" onclick="deleteNote(${index});">Delete</button></td>
+            <td><button id="dBtn" onclick="deleteNotes(${index});">Delete</button></td>
```

Comment:
"deleteButton" or "deleteBtn" would be good ID. Avoid using short ID name. "dBtn" can denote "destroy button", "doll button" or something else.

Reply:
Change everywhere if you're using this types of names in code in seperate branch.